defi/src/protocols/data2.ts

```diff
@@ -1186,7 +1186,7 @@ const data2: Protocol[] = [
     oracles: ["Chainlink"],
     forkedFrom: [],
     module: "woofi.js",
-    twitter: "WOOnetwork",
+    twitter: "_WOOFi",
```

Comment:
line 1177 description need update

Reply:
One DEX to rule all chains  - trade and earn like royalty with unmatched execution, cross-chain swaps, and single-sided yields.